src/callbox/renderer/callbox.service.ts

```diff
@@ -49,9 +51,31 @@ async function hasCurrentUserJoinedCall(token: string) {
 		throw new Error('Cannot check whether current join the call - no current user found')
 	}
 	const participants = await getCallParticipants(token)
+	if (!participants.length) {
+		throw new Error('There are currently no participants in the call', { cause: MISSED_CALL })
+	}
```

Comment:
I am not if this should be treated as an error.

Reply:
Either that or we change original method in first PR to handle 4 cases:
1) Call pending, not joined - status 200 - loop again
2) Call ended, not joined - status 201 - resolve(false) - dismiss and show 'missed call'
3) Call pending, already joined - status 404 - dismiss and show nothing
4) Other error - ?

Reply:
Replaced with `null`, should be enough for current implementation